components/ResultTable/ResultTable.js

```diff
@@ -0,0 +1,38 @@
+export default ({ players, set, matchWinner }) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          {Array.apply(null, Array(set.bestOf + 1)).map((item, i) => (
```

Comment:
Unclear what this is doing. Can you refactor into a function, or comment what is happening?

Reply:
Sure thing I can.

For clarity now, it is creating an array (length `bestOf +1` and `map`ping against it because you can't use `for` in `jsx`

Reply:
ah ok, yeah I think this would be better refactored as a function then